content/blog/coreboot-on-talos2.md

```diff
@@ -0,0 +1,307 @@
+---
+title: coreboot and Heads as an alternative firmware for OpenPOWER Talos II
```

Comment:
Because of 3mdeb and Insurgo's investment in this project and NLnet, we should mention their role. The lack of a clear message about Dasharo also worries me and defeats the purpose of this blog post. I would appreciate it if you could add 2-3 sentences that explain the role of those organizations and products.

Reply:
I can't find anything related to thin on https://nlnet.nl/project/index.html, which project was it?

Reply:
Posted nlnet exchange to piotr in DM.

Reply:
@krystian-hebel @pietrushnic @tlaurion any updates?

Reply:
There are no meaningful updates; I guess you have to mention in quite a general way that the project was extensive, took 3 years (?), and was paid through OpenCollective funds and Insurgo personal investments, as well as 3mdeb investment and NGI0 PET found.

Reply:
I opened https://github.com/linuxboot/heads/issues/1729 for Accessible Security project outcomes references

Reply:
Okay, I may add it myself. If this thread gets resolved, I will open a PR to OpenPOWER website repository.

Reply:
@krystian-hebel based on the provided link by @tlaurion, please add information about the sponsorship of the project.

Reply:
https://github.com/3mdeb/openpower-foundation-website/pull/1/commits/9958e7d9dd728dd88a3c2a00fe866a9b7b567eab

Reply:
Thank you @krystian-hebel .

@pietrushnic can we consider it resolved?